chambersd/encounters/mpg_specialization.lua

```diff
@@ -117,8 +121,13 @@ function Boss_Death(e)
   eq.spawn2(307008, 0, 0, -212, 270, 66, e.self:GetHeading()); -- NPC: Shell_of_the_Master_
 
   -- Update the Win Lockout
+  local dz = eq.get_expedition()
+  if dz.valid then
+    dz:AddLockoutDuration("Replay Timer", eq.seconds("5d")) -- 5 days + current timer (max 123 hours)
```

Comment:
Is there a reason why this call isn't just `dz:AddToReplayTimer` ? There's only one instance of a replay timer per DZ right?